test-data/golden/chat_test_msg_with_implicit_member_add

```diff
@@ -3,6 +3,6 @@ Group#Chat#10: Group chat [3 member(s)]
 Msg#10:  (Contact#Contact#11): I created a group [FRESH]
 Msg#11:  (Contact#Contact#11): Member Fiona (fiona@example.net) added by alice@example.org. [FRESH][INFO]
 Msg#12: Me (Contact#Contact#Self): You removed member Fiona (fiona@example.net). [INFO] o
-Msg#13:  (Contact#Contact#11): Member Fiona (fiona@example.net) added. [FRESH]
-Msg#14:  (Contact#Contact#11): Welcome, Fiona! [FRESH]
+Msg#13:  (Contact#Contact#11): Welcome, Fiona! [FRESH]
```

Comment:
I'd avoid changing the messages order here:
- A user might think that `Welcome, Fiona!` was sent before Fiona was added and isn't seen by Fiona.
- `Welcome, Fiona!` is more important and would be more visible if inserted in the end.